src/java/org/apache/nutch/crawl/CrawlDbReader.java

```diff
@@ -879,35 +988,37 @@ public void processTopNJob(String crawlDb, long topN, float min,
 
   }
 
-
-  public int run(String[] args) throws IOException, InterruptedException, ClassNotFoundException, Exception {
+  public int run(String[] args) throws IOException, InterruptedException,
+      ClassNotFoundException, Exception {
     @SuppressWarnings("resource")
     CrawlDbReader dbr = new CrawlDbReader();
 
     if (args.length < 2) {
-      System.err
-          .println("Usage: CrawlDbReader <crawldb> (-stats | -dump <out_dir> | -topN <nnnn> <out_dir> [<min>] | -url <url>)");
+      System.err.println(
+          "Usage: CrawlDbReader <crawldb> (-stats | -dump <out_dir> | -topN <nnnn> <out_dir> [<min>] | -url <url>)");
       System.err
           .println("\t<crawldb>\tdirectory name where crawldb is located");
       System.err
           .println("\t-stats [-sort] \tprint overall statistics to System.out");
       System.err.println("\t\t[-sort]\tlist status sorted by host");
-      System.err
-          .println("\t-dump <out_dir> [-format normal|csv|crawldb]\tdump the whole db to a text file in <out_dir>");
+      System.err.println(
+          "\t-dump <out_dir> [-format normal|csv|crawldb|json]\tdump the whole db to a text file in <out_dir>");
       System.err.println("\t\t[-format csv]\tdump in Csv format");
-      System.err
```

Comment:
Should also add a line describing `-format json`, maybe clarify that it's "JSON lines".